packages/atomic/src/components/result-template-components/atomic-result-value/atomic-result-value.tsx

```diff
@@ -37,13 +39,19 @@ export class AtomicResultValue {
         this.shouldHighlightWith!
       ) as HighlightUtils.HighlightKeyword[];
 
+      const openingDelimiter = '_openingDelimiter_';
+      const closingDelimiter = '_closingDelimiter_';
       const highlightedValue = HighlightUtils.highlightString({
         content: this.resultValue,
-        openingDelimiter: '<strong>',
-        closingDelimiter: '</strong>',
+        openingDelimiter,
+        closingDelimiter,
         highlights,
       });
-      return <Host innerHTML={highlightedValue}></Host>;
+
+      const innerHTML = sanitize(highlightedValue)
```

Comment:
Could we have instead called `sanitize` on `this.resultValue`, then highlights with `<b part="highlight"> -- </b>` ?

AKA:

```ts
 const highlightedValue = HighlightUtils.highlightString({
        content: sanitize(this.resultValue),
        '<b part="highlight">',
       '</b>,
        highlights,
});
```

Reply:
No, the highlight positions would be all messed up

Reply:
To be more clear, sanitize replace single character by multiple characters, then the highlightString method uses the position returned from the API to match the words